bump.sh

```diff
@@ -39,3 +39,24 @@ npx lerna version ${version} --yes --exact --no-git-tag-version --no-push
 
 # Generate CHANGELOG and create a commit
 npx standard-version --skip.tag=true --commit-all
+
+# Get the new version number to do some manual find and replaces
+new_version=$(node -p "require('./package.json').version")
+
+# Update the version of RFDK used in the python example
+sed -i "s/\"aws-rfdk==[0-9]*\.[0-9]*\.[0-9]*\"/\"aws-rfdk==$new_version\"/" ./examples/deadline/All-In-AWS-Infrastructure-Basic/python/setup.py
```

Comment:
This is good for the single example, but we could probably make it more general:

```bash
for exampleSetupPy in $(find ./examples/ -name 'setup.py')
do
  sed -i "s/\"aws-rfdk==[0-9]*\.[0-9]*\.[0-9]*\"/\"aws-rfdk==$new_version\"/" "$exampleSetupPy"
done
```

Reply:
Fixed

Reply:
Resolved